packages/shared/src/tableDefinitions/mapDashboards.ts

```diff
@@ -85,4 +85,10 @@ export const dashboardColumnDefinitions: UiColumnMapping[] = [
     uiTableId: "version",
     uiTableName: "Version",
   },
+  {
+    clickhouseTableName: "observations",
+    clickhouseSelect: "provided_model_name",
+    uiTableId: "model",
+    uiTableName: "Model",
```

Comment:
can all charts deal with this filter condition?

Reply:
From what I can see this is only used in the query builder and not in the offered filter options on the dasboard page: 

![CleanShot 2024-12-16 at 15 54 54@2x](https://github.com/user-attachments/assets/0ac3cc64-d254-4cdd-a7d4-2e64e74cfc08)

Am I missing something maybe?